BE/GovernmentEntities/AsianJurisdiction/EasternAsiaGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -107,35 +100,30 @@
 	<owl:NamedIndividual rdf:about="&fibo-be-ge-easj;GovernmentOfThePeoplesRepublicOfChina">
 		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
 		<rdfs:label xml:lang="en">Government of the People&apos;s Republic of China</rdfs:label>
+		<rdfs:label xml:lang="zh">中华人民共和国</rdfs:label>
 		<skos:definition>unitary Marxist-Leninist single-party socialist republic in Eastern Asia, covering approximately 9.6 million square kilometers (3.7 million square miles), that is the world&apos;s third or fourth-largest country by area</skos:definition>
 		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-easj;JurisdictionOfChina"/>
 		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;China"/>
 	</owl:NamedIndividual>
 	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-easj;GovernmentOfTheRepublicOfChina">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of the Republic of China</rdfs:label>
+		<rdfs:label xml:lang="zh">中華民國</rdfs:label>
+		<skos:definition>unitary semi-presidential constitutional republic whose neighbors include the People&apos;s Republic of China (PRC) to the northwest, Japan to the northeast, and the Philippines to the south</skos:definition>
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-easj;JurisdictionOfTheRepublicOfChina"/>
```

Comment:
There is no reason for our URLs to be so confusing between China and Taiwan! Why not use Taiwan for all URLs except the official country legal name?